tests/mce_check/mce_check.cpp

```diff
@@ -106,7 +106,7 @@ struct test mce_test = {  // This variable is used in the framework!
         .test_run = mce_check_run,
         .desired_duration = -1,
         .fracture_loop_count = -1,
-        .quality_level = INT_MAX,
+        .quality_level = test_quality(INT_MAX),
```

Comment:
Let's just use `TEST_QUALITY_PROD` here instead of casting it to enum.

Reply:
Sure.